### PR TITLE
Check if user_info is an array before counting

### PR DESCRIPTION
**Pull request by nupplaphil**
Fixing https://github.com/friendica/friendica/issues/6337#issuecomment-468839548

### CODE REVIEW

**Comment by MrPetovan** [comment]
I have a much larger fix/rewrite to submit, I guess I'm going to submit it to `develop` now. 😅

File: include/api.php [hunk 1]
```diff
@@ -953,7 +953,7 @@ function api_account_verify_credentials($type)
 	// - Adding last status
 	if (!$skip_status) {
 		$user_info["status"] = api_status_show("raw");
-		if (!count($user_info["status"])) {
+		if (!is_array($user_info["status"]) || !count($user_info["status"])) {
```
Comment (MrPetovan) [inline include/api.php]: `api_status_show('raw')` can return void if there aren't any last public message. You'd avoid an issue with the count but you'd get an error with `unset($user_info["status"]["user"]);` on line 959.

**Comment by nupplaphil** [comment]
> I have a much larger fix/rewrite to submit, I guess I'm going to submit it to `develop` now. sweat_smile

I'm feeling with you ;-)

File: include/api.php (hunk 1 above)
Comment (nupplaphil) [reply include/api.php]: I see. That's a code smell too I think, because we're mixing return and not returning in one function.

**Pull request merged**